src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -60,4 +63,13 @@ public void updateDashboard() {
         SmartDashboard.putNumber("Y", gyro.getYaw());
         SmartDashboard.putNumber("Z", gyro.getRoll());
     }
+
+    public static Drivetrain create() {
+        Spark LF = new Spark(RobotMap.PWM.FrontMotorLeft.getPort());
+        Spark RF = new Spark(RobotMap.PWM.FrontMotorRight.getPort());
+        Spark LB = new Spark (RobotMap.PWM.BackMotorLeft.getPort());
+        Spark RB = new Spark (RobotMap.PWM.BackMotorRight.getPort());
+
+        return new Drivetrain(List.asList(RF,LF), motorR, encoderL, encoderR, gyro)
```

Comment:
motorL needs a list of the left motors and motorR needs a list of the right motors